scalyr_agent/scalyr_client.py

```diff
@@ -718,8 +719,9 @@ def __receive_response(self, body_str, send_time):
                 elif status == "error/client/badParam":
                     log.error(
                         "Request to '%s' failed due to a bad parameter value.  This may be caused by an "
-                        "invalid write logs api key in the configuration",
+                        "invalid write logs api key in the configuration. Response message: %s",
                         self.__full_address,
+                        json.loads(response).get("message"),
```

Comment:
It seems like ``response_as_json`` variable should already be available here. We should probably use that (this way we can avoid parsing response twice and the need to wrap this block in try / catch in case parsing fails).

 Adding a test case would also be good.